akd_test_tools/src/fixture_generator/examples/test.yaml

```diff
@@ -30,1239 +30,2205 @@ epoch: 9
 records:
   - TreeNode:
       label:
-        val: 4D8FE7E924913420EA65BB44DD7CBA138D0C3DA3A77EDBF342CAEF1E71EEBB11
-        len: 256
-      last_epoch: 1
-      least_descendent_ep: 1
-      parent:
-        val: "4000000000000000000000000000000000000000000000000000000000000000"
-        len: 2
-      node_type: Leaf
-      left_child: ~
-      right_child: ~
-      hash: DF57D5005E8E700F6EDD67228BBCE452DC93B5635B8F355759B474041F3D4C79
-  - TreeNode:
-      label:
-        val: F0095EEAD1AB85170C2D572A4E257AB36043CA6BD2444A983584AFAAC92B703E
-        len: 256
-      last_epoch: 4
-      least_descendent_ep: 4
-      parent:
-        val: F000000000000000000000000000000000000000000000000000000000000000
-        len: 7
-      node_type: Leaf
-      left_child: ~
-      right_child: ~
-      hash: D83CA2E6CA89C340ADE2B533CCAA2738D94AF033C703E14A419EBBDDE4E748EF
+        val: 61B6CF9432424B3139006A631373FA7C62226FEBCD8942868BC2EA1812C3919A
```

Comment:
We could forget updating this file. Should we keep it checked in or somehow auto-build?

Reply:
Ideally changes to the Storage layer should be accompanied with a minor version bump, which the [`akd_test_tools` crate specifically depends on](https://github.com/novifinancial/akd/blob/7958a46989b5e5d4f633e645a47fff0945350d85/akd_test_tools/Cargo.toml#L24). So long as the person updating `akd_test_tool`'s Cargo file checks for Storage layer changes and re-generates this file accordingly, it should never become "incorrect" for a given akd version.

Reply:
I think I'm going to roll to something other than 0.6.X (i.e. bump to 0.7) since we're doing a pretty big storage contract break here. But yes a version bump was on my radar, thanks for reminding me!

Reply:
It's possible to glue a few Github actions together to re-gen this file if there is a change in the storage layer. Let me know whether folks would be interested in such an automated solution.